### ISSUE TITLE
Places to Create Git Repositories execrise is poorly worded

### ISSUE THREAD

**Issue by novica** [swcarpentry/git-novice#1028]
### How could the content be improved?

It seems the point of this execrise is to teach that you could accidentaly do `git init` twice in the same directory and thus checking for `status` is good practice. 

However it is factually incorrect.

1. Git does not track folders. So when you say "repository will track all files, sub-directories, and subdirectory" this is not correct. Git ignores empty folders. It will not automatically track an empty folder. Maybe there should be a point explaining this

3. Git repositories can be nested https://git-scm.com/book/en/v2/Git-Tools-Submodules  (although this is an advanced topic). So, this is not correct: "Git repositories can interfere with each other if they are “nested”".

### Which part of the content does your suggestion apply to?

https://swcarpentry.github.io/git-novice/instructor/03-create.html

**Comment by martinosorb** [comment]
Thanks for the observation.

Regarding point 1., you are right, although I think this is a relatively minor detail since we are at the very beginning. But a short sentence, or simply saying 'non-empty subdirectories', could be done.
Regarding point 2., I would say it is true that they interfere: the point is precisely to say that they can be nested, and this brings all sorts of advanced topics (and problems, for the novice). I think at this stage, it is best to recommend not doing it at all.

Do you have any suggestion for small changes to this effect?

**Comment by froggleston** [comment]
I would go one step further and suggest:

`Alfredo does not need to make the desserts subdirectory a Git repository because the recipes repository will track all files under the recipes directory, including those in any subdirectories. Note that Git does not track empty directories.`

The .gitkeep idea is a nice one and fits with convention, but does presuppose that people know the dot syntax for hidden Linux files. Would this need an explicit mention?

**Comment by novica** [comment]
Yeah true. Bu there is a .gitignore section in the lesson as well. So hidden files get more mentions:)

**Comment by froggleston** [comment]
Fair point!

**Comment by novica** [comment]
I read over the text a few times, and the way @froggleston phrased that paragraph makes more sense to me.

I think the question should be also amended to include a file creation. Otherwise, the point of not tracking empty directories will be mute. 

If I do the two exercises in sequence, after I remove the `.git` folder in the subdirectory, `git status` won't tell me that I have a folder to add to my project, because the folder would be empty.

**Comment by erinmgraham** [comment]
Just taking another look at this: there is a callout in the next episode, 04-changes.html | Directories | Two important facts you should know about directories in Git. The first fact talks about empty directories, then creates cake files, then commits the change (which we don't with desserts). 

Maybe we can make one challenge from both of these examples in just one of the episodes? I would favour ep 03 with cakes and committing, which also meets @novica point about file creation.

If we leave the two separate, I'd favor changing the solution in episode 03-create to froggleston's two sentence answer.
